build.gradle

```diff
@@ -1,30 +1,51 @@
+import org.springframework.boot.gradle.plugin.SpringBootPlugin
+
+import java.time.Instant
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
 plugins {
-    id 'org.springframework.boot' version '2.1.5.RELEASE'
     id 'java'
+    id 'org.springframework.boot' version '2.1.9.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.8.RELEASE'
```

Comment:
`id 'com.github.ben-manes.versions' version '0.26.0'` worth adding in as a plugin so you can then run:
```./gradlew dependencyUpdates``` to find out if any of your libraries need updating

Reply:
nice